modules/sdk/gradle-plugins-xsd-builder/README.markdown

```diff
@@ -0,0 +1,100 @@
+# XSD Builder Gradle Plugin
+
+The XSD Builder Gradle plugin allows you to generate [Apache XMLBeans](https://xmlbeans.apache.org/)
+bindings from XML Schema (XSD) files.
+
+## Usage
+
+To use the plugin, include it in your build script:
+
+```gradle
+buildscript {
+	dependencies {
+		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.xsd.builder", version: "1.0.5"
+	}
+
+	repositories {
+		maven {
+			url "https://cdn.lfrs.sl/repository.liferay.com/nexus/content/groups/public"
+		}
+	}
+}
+
+apply plugin: "com.liferay.xsd.builder"
+```
+
+The XSD Builder plugin automatically applies the [`java`](https://docs.gradle.org/current/userguide/java_plugin.html)
+plugin.
+
+The plugin automatically resolves the Apache XMLBeans library as a dependency,
+therefore, you have to configure a repository hosting the library and its
+transitive dependencies. One repository that hosts them all is the Liferay CDN:
```

Comment:
Hi Andrea. Are you suggesting the developer specify the Liferay CDN as a repository because it hosts the XMLBeans library and its transitive dependencies?